libs/libical/Makefile

```diff
@@ -8,11 +8,11 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libical
-PKG_VERSION:=3.0.9
+PKG_VERSION:=3.0.11
 PKG_RELEASE:=2
```

Comment:
```suggestion
PKG_RELEASE:=1
```

Reply:
all the tests succeeded, it is only because it is not signed with my full name? strange reason? I don't want my name out there, but my mail already revealed my name :(